uber/templates/base.html

```diff
@@ -90,7 +90,13 @@
             }
             $('.footable').footable();
             $('.date').datetextentry({
-                field_order: 'MDY'
+                field_order: 'MDY',
+                min_year : '1890',
```

Comment:
Haha, I probably would have just said 1900 since it's a nice round number, but I like the idea that we were concerned about 116-year-old MAGFest attendees.